python_modules/dagster-graphql/dagster_graphql/schema/sensors.py

```diff
@@ -73,6 +73,7 @@ class GrapheneSensor(graphene.ObjectType):
     nextTick = graphene.Field(GrapheneDryRunInstigationTick)
     metadata = graphene.NonNull(GrapheneSensorMetadata)
     sensorType = graphene.NonNull(GrapheneSensorType)
+    assetSelection = graphene.String()
```

Comment:
should this be an graphql object with like a description or name field to give us some design space to include more structured info in the future? e.g. if we wanted to include the list of asset keys

Reply:
👍 made this change